internal/state/state_test.go

```diff
@@ -858,80 +830,21 @@ func TestFourAddFourMinusOneGenesisValidators(t *testing.T) {
 	// add 10 validators with the same voting power as the one added directly after genesis:
 	for i := 0; i < 10; i++ {
 		ld := llmq.MustGenerate(append(proTxHashes, crypto.RandProTxHash()))
-		abciValidatorUpdates := make([]abci.ValidatorUpdate, len(ld.ProTxHashes))
-		for j, proTxHash := range ld.ProTxHashes {
-			abciValidatorUpdates[j] = abci.UpdateValidator(
-				proTxHash,
-				ld.PubKeyShares[j].Bytes(),
-				types.DefaultDashVotingPower,
-				types.RandValidatorAddress().String(),
-			)
-		}
-		abciThresholdPublicKey3, err := cryptoenc.PubKeyToProto(ld.ThresholdPubKey)
-		assert.NoError(t, err)
-		abciValidatorSetUpdate := abci.ValidatorSetUpdate{
-			ValidatorUpdates:   abciValidatorUpdates,
-			ThresholdPublicKey: abciThresholdPublicKey3,
-			QuorumHash:         quorumHash,
-		}
-
-		validatorUpdates, thresholdPublicKey3, _, err :=
-			types.PB2TM.ValidatorUpdatesFromValidatorSet(&abciValidatorSetUpdate)
-		assert.NoError(t, err)
-
-		abciResponses := &tmstate.ABCIResponses{
-			BeginBlock: &abci.ResponseBeginBlock{},
-			EndBlock:   &abci.ResponseEndBlock{ValidatorSetUpdate: &abciValidatorSetUpdate},
-		}
-		block, err := statefactory.MakeBlock(oldState, oldState.LastBlockHeight+1, new(types.Commit), nil, 0)
-		require.NoError(t, err)
-		blockID := types.BlockID{Hash: block.Hash(), PartSetHeader: block.MakePartSet(testPartSize).Header()}
-		state, err = sm.UpdateState(state, firstNodeProTxHash, blockID, &block.Header, abciResponses,
-			validatorUpdates, thresholdPublicKey3, quorumHash)
+		abciValidatorSetUpdate, err := abci.LLMQToValidatorSetProto(*ld)
```

Comment:
this adds validators one by one, we should also test change of multiple validators at once